ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -638,6 +659,14 @@ def runTest(self):
         self.log("Schedule to reboot the remote switch in %s sec" % self.reboot_delay)
         thr.start()
 
+        self.log("Wait until DUT reboots")
+        self.timeout(self.task_timeout, "DUT hasn't rebooted in %d seconds" % self.task_timeout)
+        self.wait_until_dut_reboots()
+        self.cancel_timeout()
+
+        self.reboot_start = datetime.datetime.now()
+        self.log("Dut reboots: reboot start %s" % str(self.reboot_start))
+
```

Comment:
I'd suggest to insert a check that the DUT data plane is still working.

Right after you found your control plane are rebooting, check that your DUT is able to send packets back and forth. And after that wait until DUT data plane is stopped working.